kedro-airflow/setup.py

```diff
@@ -16,11 +16,6 @@
 with open("requirements.txt", "r", encoding="utf-8") as f:
     requires = [x.strip() for x in f if x.strip()]
 
-# get test dependencies and installs
-with open("test_requirements.txt", "r", encoding="utf-8") as f:
-    test_requires = [x.strip() for x in f if x.strip() and not x.startswith("-r")]
-
-
 # Get the long description from the README file
```

Comment:
```suggestion
# get the long description from the README file
```
Nit: Just because there's two instances starting with lowercase "get", and I find it annoying, if you don't mind rolling it in. 🙏

Reply:
done